doc/nrf/gs_programming.rst

```diff
@@ -143,7 +143,7 @@ If it is installed, verify that its location is correct in the PATH variable or,
 Building on the command line
 ****************************
 
-Complete the following steps to build |NCS| projects on the command line after completing the :ref:`command-line build setup <build_environment_cli>`.
+After completing the :ref:`command-line build setup <build_environment_cli>`, use the following steps to build |NCS| projects on the command line.
```

Comment:
The link goes to _Installing manually_, but the user could also have followed _Installing automatically_, and that would be fine. Would it be possible to communicate both somehow? How about splitting the sentence, so the first part would be a note: "First complete the steps  to install the toolchain _automatically_ or _manually_". Then after the note: "Use the following steps...".